tests/integration/system/test_system_client.py

```diff
@@ -0,0 +1,259 @@
+        tgt=tgt,
+        fun=fun_2,
+        metadata=metadata,
+    )
+    responses.append(create_job(job_2))
+
+    return responses
+
+
+@pytest.mark.integration
+@pytest.mark.enterprise
+class TestSystemClient:
+    def test__create_job__succeeds(
+        self,
+        create_job,
+    ):
+        arg = [["A description"]]
+        tgt = [
+            "HVM_domU--SN-ec200972-eeca-062e-5bf5-017a25451b39--MAC-0A-E1-20-D6-96-2B"
+        ]
+        fun = ["system.set_computer_desc"]
+        metadata = {"queued": True, "refresh_minion_cache": {"grains": True}}
+        job = CreateJobRequest(
+            arg=arg,
+            tgt=tgt,
+            fun=fun,
+            metadata=metadata,
+        )
+
+        response = create_job(job)
+
+        assert response is not None
+        assert response.jid is not None
+        assert response.arg == arg
+        assert response.tgt == tgt
+        assert response.metadata == metadata
+        assert response.fun == fun
+
+    def test__list_jobs__single_job__succeeds(self, create_job, client: SystemClient):
+        arg = [["A description"]]
+        tgt = [
+            "HVM_domU--SN-ec200972-eeca-062e-5bf5-017a25451b39--MAC-0A-E1-20-D6-96-2B"
+        ]
+        fun = ["system.set_computer_desc"]
+        metadata = {"queued": True, "refresh_minion_cache": {"grains": True}}
+        job = CreateJobRequest(
+            arg=arg,
+            tgt=tgt,
+            fun=fun,
+            metadata=metadata,
+        )
+        create_job_response = create_job(job)
+
+        response = client.list_jobs(jid=create_job_response.jid)
+        assert response is not None
+        assert len(response) == 1
+
+        [response_job] = response
+
+        assert response_job.jid == create_job_response.jid
+        assert response_job.config is not None
+        assert response_job.config.arg == arg
+        assert response_job.config.tgt == tgt
+        assert response_job.metadata == metadata
+        assert response_job.config.fun == fun
+
+    def test__list_jobs__multiple_jobs__succeeds(
+        self, create_multiple_jobs, client: SystemClient
+    ):
+        response = client.list_jobs(system_id=create_multiple_jobs[0].tgt[0])
+        assert response is not None
+        assert len(response) == 2
+
+    def test__list_jobs__multiple_jobs_take_one__succeeds(
+        self, create_multiple_jobs, client: SystemClient
+    ):
+        response = client.list_jobs(system_id=create_multiple_jobs[0].tgt[0], take=1)
+        assert response is not None
+        assert len(response) == 1
+
+    def test__list_jobs__multiple_jobs_skip_one__succeeds(
+        self, create_multiple_jobs, client: SystemClient
+    ):
+        response = client.list_jobs(system_id=create_multiple_jobs[0].tgt[0], skip=1)
+        assert response is not None
+        assert len(response) == 1
+
+    def test__list_jobs__Invalid_system_id__fails(self, client: SystemClient):
+        with pytest.raises(Exception):
```

Comment:
```suggestion
        with pytest.raises(ApiException):
```
reflect this in other places.

Reply:
Modified & reflected in other places.